nilearn/_utils/fixes/funcsigs.py

```diff
@@ -0,0 +1,814 @@
+# Copyright 2001-2013 Python Software Foundation; All Rights Reserved
```

Comment:
Why is this file added?

Are you sure that it's not the result of an operation error with git?